src/utils/inputs.ts

```diff
@@ -6,8 +6,42 @@ export const checkSum = core.getInput("checksum");
 export const enableCache = core.getInput("enable-cache") === "true";
 export const cacheSuffix = core.getInput("cache-suffix") || "";
 export const cacheLocalPath = getCacheLocalPath();
-export const githubToken = core.getInput("github-token");
 export const cacheDependencyGlob = core.getInput("cache-dependency-glob");
+export const toolBinDir = getToolBinDir();
+export const toolDir = getToolDir();
+export const githubToken = core.getInput("github-token");
+
+function getToolBinDir(): string | undefined {
+  const toolBinDirInput = core.getInput("tool-bin-dir");
+  if (toolBinDirInput !== "") {
+    return toolBinDirInput;
+  }
+  if (process.platform === "win32") {
+    if (process.env.RUNNER_TEMP !== undefined) {
+      return `${process.env.RUNNER_TEMP}${path.sep}uv-tool-bin-dir`;
+    }
+    throw Error(
+      "Could not determine UV_TOOL_BIN_DIR. Please make sure RUNNER_TEMP is set or provide the tool-bin-dir input",
```

Comment:
Is it that `RUNNER_TEMP` is always defined on GitHub runners?

Reply:
So tests show that it is and I could not find contrary information